src/app.ts

```diff
@@ -75,3 +76,5 @@ export class Client {
 		});
 	}
 }
+
+console.log("Working again");
```

Comment:
Avoid debug logs. They are okay for debugging but should not be added in commits. It doesn't look good for us if end customer is using our SDK and sees message like this in his console. If logging in necessary then print meaningful message like "MOK: SDK client loaded." or something like that.